### PR TITLE
fix(jetty): upgrade jetty dependency for CVE

### PR DESCRIPTION
**Pull request by RyanHolstien**
Upgrade Jetty to fix CVEs. Previously we were on Jetty 9.4.20 which had the following CVE: https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2021-28165

### CODE REVIEW

**Comment by RyanHolstien** [comment]
Only frontend tests are failing despite this change being exclusive to GMS. Will investigate further, but note that I was able to duplicate the Policies issue on this branch so it is the likely culprit.

File: docker/datahub-gms/Dockerfile [hunk 1]
```diff
@@ -15,9 +15,9 @@ RUN apk --no-cache --update-cache --available upgrade \
       echo >&2 "Unsupported architecture $(arch)" ; exit 1; \
     fi \
     && apk --no-cache add tar curl openjdk8-jre bash coreutils gcompat \
-    && curl https://repo1.maven.org/maven2/org/eclipse/jetty/jetty-runner/9.4.20.v20190813/jetty-runner-9.4.20.v20190813.jar --output jetty-runner.jar \
-    && curl https://repo1.maven.org/maven2/org/eclipse/jetty/jetty-jmx/9.4.20.v20190813/jetty-jmx-9.4.20.v20190813.jar --output jetty-jmx.jar \
-    && curl https://repo1.maven.org/maven2/org/eclipse/jetty/jetty-util/9.4.20.v20190813/jetty-util-9.4.20.v20190813.jar --output jetty-util.jar \
+    && curl https://repo1.maven.org/maven2/org/eclipse/jetty/jetty-runner/9.4.46.v20220331/jetty-runner-9.4.46.v20220331.jar --output jetty-runner.jar \
```
Comment (rslanka) [inline docker/datahub-gms/Dockerfile:18]: How about a similar update in docker/datahub-ingestion/Dockerfile?

Comment (RyanHolstien) [reply docker/datahub-gms/Dockerfile:18]: I can update the comment, but it doesn't actually use it from what I can tell?

**Review by shirshanka — APPROVED** [review]
Approving. Failures don't seem related.

**Pull request merged**